sdk/monitor/Azure.Monitor.Query/tests/LogsTestData.cs

```diff
@@ -134,5 +136,25 @@ private async Task<int> QueryCount(string workspaceId)
                 return 0;
             }
         }
+
+        private async Task InitializeStorageAccount()
+        {
+            var client = new LogsQueryClient(_testEnvironment.Credential);
+
+            while (true)
+            {
+                var result = await client.QueryResourceAsync(new ResourceIdentifier(_testEnvironment.StorageAccountId), "search *", new QueryTimeRange(DateTimeOffset.Now, TimeSpan.FromDays(15))).ConfigureAwait(false);
+                if (result.Value.Table.Rows.Count > 0 || result.Value.Table.Columns.Count > 0)
```

Comment:
I thought we needed both rows and columns to be greater than 0?

Reply:
yes, this should be an and, will update